src/sentry/snuba/metrics/fields/base.py

```diff
@@ -1427,25 +1438,25 @@ def generate_where_statements(
                 SessionMRI.ERRORED_USER_ALL.value,
             ],
             unit="users",
-            snql=lambda all_user_count, errored_user_all_count, org_id, metric_ids, alias=None: subtraction(
+            snql=lambda all_user_count, errored_user_all_count, project_ids, org_id, metric_ids, alias=None: subtraction(
                 all_user_count, errored_user_all_count, alias=alias
             ),
             post_query_func=lambda *args: max(0, *args),
         ),
         SingularEntityDerivedMetric(
             metric_mri=TransactionMRI.ALL.value,
-            metrics=[TransactionMRI.DURATION.value],
+            metrics=[TransactionMRI.DURATION.value, TransactionMRI.MEASUREMENTS_LCP.value],
```

Comment:
The `LCP` is needed because now the query uses a `multiIf` in which we either use `duration` and `lcp` as found here -> https://github.com/getsentry/sentry/blob/b468ff4a9049f046172eb53ad6b11deeaa4ab77f/src/sentry/snuba/metrics/fields/snql.py#L274-L275

Reply:
The need for this is because the `snql` function will be provided with all the ids of its constituent metrics which in this case can be either `LCP` or `DURATION`.